contribs/gmf/examples/mobilebackgroundlayerselector.html

```diff
@@ -56,9 +56,9 @@
     directive to select the map background layer.</p>
     <div>
         <span>Select base layer:</span>
-        <gmf-mobile-background-layer-selector
-          gmf-mobile-background-layer-selector-map="::ctrl.map">
-        </gmf-mobile-background-layer-selector>
+        <gmf-mobilebackgroundlayerselector
```

Comment:
As a German I am used to long words, but `mobilebackgroundlayerselector` is totally unreadable for me. ;)

Camel-case or separating with "-" helps a lot!